Fiscalizer.md

```diff
@@ -0,0 +1,98 @@
+## Fiscalizer
+
+We are using the [fiscalizer gem](https://github.com/infinum/fiscalizer) to transfer various document types from a business entity to the tax authorities.
+
+### Setup & Usage
+
+Everything about setup and usage of this gem is explained on [fiscalizer gem](https://github.com/infinum/fiscalizer).
+
+### Certificates
+
+#### Environments:
+
++ #### Production environment
+
+Requires only one certificate - application certificate. It is most often called `FISCAL_1.p12`. It contains the public and private key, and the CA (`Certificate authority`) certificate.
+
+Secrets file example:
+
+```Ruby
+fiscalization_app_cert_path: 'cert/production/FISKAL_1.p12'
+fiscalization_ca_cert_path: ''
+```
+
+
++ #### Development & Staging environment
+
+Requires 2 certificates - both application and server certificate. The application certificate (`FISCAL_1.p12`) contains the public and private key. The server certificate (most often `fina_ca.pem`) contains the required CA certificates.
+
+Secrets file example:
+
+```Ruby
+fiscalization_app_cert_path: 'cert/development/FISKAL_1.p12'
+fiscalization_ca_cert_path: 'cert/development/fina_ca.pem'
+```
+
++ #### Application Certificate unpacking
+
+To view the content of the application certificate locally, you need to unpack it with this command:
+
+```
+openssl pkcs12 -in FISKAL_1.p12 -out FISKAL_1.pem -nodes
+```
+
+After this command, you are prompted to input the `fiscalization password`.
+
++ #### Updating from 0.x to 1.x
```

Comment:
is this inside readme of fiscalizer? if so i would not put it here

Reply:
@d4be4st noup, there's nothing about updating in the fiscalizer readme. I could put it there tho because it has more sense for that to be on the gem readme, and then it wouldn't be here anymore. go so?

Reply:
yes,

I wouldn't even mention it here actually, because all our project actually use 1.x version now